gsa/src/web/components/bar/menubar.js

```diff
@@ -136,6 +137,15 @@ const MenuBar = ({
                   to="vulnerabilities"
                 />
               }
+              <MenuSection>
+                {capabilities.mayAccess('tickets') &&
+                  <MenuEntry
+                    section
+                    title={_('Remediation Tickets')}
```

Comment:
Tickets in the rest of GSA are simply called tickets (e.g. in the trashcan), and not Remediation Tickets. To avoid confusion I'd adjust that to be consistent. It would be easiest to do it here and worst to do it in all other instances. Doing it here would against the FS, though.